packages/integrations/useMark/index.ts

```diff
@@ -0,0 +1,81 @@
+import { computed, ref } from 'vue-demi'
+import type { Ref } from 'vue-demi'

```

Comment:
import type is best placed at the bottom of all references.

Reply:
I've moved the type imports under the rest of the imports, is that what you meant?